widgets/SapOData-batchsource.json

```diff
@@ -0,0 +1,91 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "display-name": "SAP OData",
+  "configuration-groups": [
+    {
+      "label": "General",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Reference Name",
+          "name": "referenceName"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "OData Service URL",
+          "name": "url"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Resource Path",
+          "name": "resourcePath",
+          "plugin-function": {
+            "label": "Get Schema",
+            "widget": "outputSchema",
+            "output-property": "schema",
+            "omit-properties": [
+              {
+                "name": "schema"
+              }
+            ],
+            "add-properties": [
```

Comment:
is this needed?

Reply:
An OData query can contain '$select' option. The $select option specifies a subset of properties to include in the response body. For example, to get only the name and price of each product, the following query can be used: 'http://localhost/odata/Products?$select=Price,Name'.

Thus, we should use this query option to include only needed fields in the inferred schema. 

Seems that it's convenient to have an ability to modify the query in the "Get Schema" modal(just like with database plugins).

Reply:
Query is already a config property though, so we don't need to ask the user for it again. We can just use whatever they have already given.

The DB plugins were a bad example, they should just use whatever is given in the import query and not require the user to use a separate query that is actually expected to be the same as the import query.

Reply:
Ok, thanks. Removed.